test/nuts/delete.nut.ts

```diff
@@ -12,6 +12,16 @@ import { expect } from 'chai';
 import { execCmd } from '@salesforce/cli-plugins-testkit';
 import { SourceTestkit } from '@salesforce/source-testkit';
 import { exec } from 'shelljs';
+import { FileResponse } from '@salesforce/source-deploy-retrieve';
+
+const queryOrgAndFS = (name: string, fsPath: string): void => {
+  const cmd = `force:data:soql:query -q "SELECT IsNameObsolete FROM SourceMember WHERE MemberType='LightningComponentBundle' AND MemberName='${name}'" -t --json`;
```

Comment:
suggestion: get a connection from the setup org:create step, similar to what you did in the `deployDestructive.nut` 

```ts
conn = await Connection.create({
      authInfo: await AuthInfo.create({
        username: (session.setup[0] as { result: { username: string } }).result?.username,
      }),
    });
```

so it doesn't create another cross-plugin dependency